### ISSUE TITLE
Sealed Simulator - Opening large amount of boosters will fail

### ISSUE THREAD

**Issue by rosstroha** [dantroha/mtg-mirror#3]
This is an extreme use case since few people will be trying to open 90,000 booster packs, therefore it is low priority. 

The input box has no limit for the number boosters and so the `openBoosters()` function fails with this JS error:

``` Javascript
vendor.min.js:2 RangeError: Maximum call stack size exceeded
    at combineCardArrays (app.min.js:formatted:887)
    at Object.openBoostersForCardSetGroups (app.min.js:formatted:859)
    at sealedsim.vm.openBoosters (app.min.js:formatted:616)
    at fn (eval at compile (vendor.min.js:1), <anonymous>:4:262)
    at b (vendor.min.js:2)
    at e (vendor.min.js:5)
    at m.$eval (vendor.min.js:3)
    at m.$apply (vendor.min.js:3)
    at HTMLButtonElement.<anonymous> (vendor.min.js:5)
    at Sf (vendor.min.js:1)
```

Fix by limiting the number of boosters that are allowed to be opened. This can be set with an html attribute `max=x`. Also add a condition somewhere in the JavaScript. According to [this Stackoverflow comment](http://stackoverflow.com/questions/22123769/rangeerror-maximum-call-stack-size-exceeded-why#comment33565591_22123769) the limit for an array is 65536.

@dantroha What would be a reasonable limit to set?

**Comment by dantroha** [comment]
Yeah like 1,000 or 9,999 is a ridiculously high number and I can't see much reason to go over that amount.
